### PR TITLE
Form 21-8940: Ability to provide Certification#14543

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
Form 21-8940: Ability to provide Certification

As a Veteran submitting a claim for Individual Unemployability on the Vets.gov platform, I need the ability to provide Certification to the veracity of the information provided, so that I can provide additional evidence associated to my Individual Unemployability while completing the Form 21-8940 application interview.

## Testing done
- [x] local testing
- [x] e2e tests

## Screenshots
screen shot 2018-12-12 at 11 55 16 am

## Acceptance criteria
- [x] The user is able to certify the information provided { check box, REQUIRED FIELD }
- [x] The user is able to certify statement provided { check box, REQUIRED FIELD } 
- [x] The user is only able to move to the next step in the process when both statements are certified, Story#TBD
- [x] Field validations (if applicable per the linked document for Validations above) are enforced in the UI
- [x] Error messages (if applicable per the linked document for Validations above) are displayed in the UI

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/tests/pages/unemployabilityCertification.unit.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      true,
+    );
+
+    selectCheckbox(
+      form,
+      'root_unemployability_view:informOfReturnToWork_view:informOfReturnToWorkAccept',
+      true,
+    );
+
+    form.find('form').simulate('submit');
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/tests/pages/unemployabilityCertification.unit.spec.jsx]: 😉

File: src/applications/disability-benefits/all-claims/pages/unemployabilityCertification.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        },
+      ],
+    },
+  },
+};
+
+export const schema = {
+  type: 'object',
+  properties: {
+    unemployability: {
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/pages/unemployabilityCertification.js]: I'm wondering if we should follow this method throughout all of 8940 questions (at least the ones related to the schema). Maybe bring this up with the team so we're all on the same page.

File: src/applications/disability-benefits/all-claims/pages/unemployabilityCertification.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      type: 'object',
+      properties: {
+        'view:statementsAreTrue': {
+          type: 'object',
+          required: ['view:statementsAreTrueAccept'],
+          properties: {
+            'view:statementsAreTrueAccept': {
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/pages/unemployabilityCertification.js]: Do we know if any of these need to make it to the backend? If so we might need to remove `view:`

File: src/applications/disability-benefits/all-claims/config/form.js [hunk 4]
```diff
@@ -384,6 +385,14 @@ const formConfig = {
           uiSchema: hospitalizationHistory.uiSchema,
           schema: hospitalizationHistory.schema,
         },
+        unemployabilityCertification: {
+          title: 'Certification',
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/config/form.js]: This title shows up on the review page, maybe use something more related to unemployability?

**Review by aaronjlech — COMMENTED** [review]
Just a few questions around the schema everything else looks good!

**Review by aaronjlech — APPROVED** [review]
lgtm!

**Comment by zurbergram** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group - Could I get a code review on this at your earliest convenience? Thanks!

**Review by rianfowler — APPROVED** [review]

**Review by jbalboni — APPROVED** [review]

**Pull request merged**